backend/uchicago/watcher.py

```diff
@@ -0,0 +1,67 @@
+import requests
+from lxml import html
+import pyrebase
+from difflib import Differ
+import sendgrid
+import os
+from sendgrid.helpers.mail import *
+
+FIREBASE = pyrebase.initialize_app({
+    'apiKey':
+    'AIzaSyCjBDyhwbXcp9kEIA2pMHLDGxmCM4Sn6Eg',
+    'authDomain':
+    'canigraduate-43286.firebaseapp.com',
+    'databaseURL':
+    'https://canigraduate-43286.firebaseio.com',
+    'storageBucket':
+    'canigraduate-43286.appspot.com',
+    'serviceAccount':
+    './service_account_key.json'
+})
+
+def watcher(program):
+	url = db.child('programs').child(program).child('metadata').child('catalog').get().val()
```

Comment:
Can you run yapf on this file? It should correct all the formatting issues that matter.

Reply:
I just did.